.editorconfig

```diff
@@ -5,4 +5,4 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Comment:
2 spaces indent in Java would be hard to read, consider changing it for YAML only

Reply:
:+1: